### ISSUE TITLE
Auto-publishing of SQL request results to a repository of data exports in CSV

### ISSUE THREAD

**Issue by xekc** [Lokiedu/libertysoil-site#841]
We plan to publish site's data as OpenData automatically in a number of small datasets. 

At this point we need proof-of-concept implementation to showcase the method itself and have some first bits of live data actually published. Further on we'll expand on number of database data segments that get published - and on the number of output formats too. 

Our goal is to make information accessible and usable for tools that non-profits and social change organisations would use. We'll start with CSV (the easiest one?) but it's quite likely we'll expand the list very soon looking at data analysis tools people that aren't technically savvy would use for their work.

## Technical Details

For the MVP we will need to create a `bin/opendata-csv.js` script which will:

1. Connect to database and select information about all of the schools
2. Create/update `schools.csv` file in a directory specified by `OPENDATA_REPO` environment variable
3. Use [NodeGit](http://www.nodegit.org) if current dump differs from previously committed version. Makes a "commit" and "push" if it does.

Additionally, we will create a periodic task definition for our server environment (not part of this repository) which will run this script on schedule.

**Comment by xekc** [comment]
@indeyets I'm expecting this to work soon on a proof-of-concept stage and then expand to include additional segments of data (that should be as long as modifying / writing those SQL requests, right?). I want to add various perspectives that we'll rely on, some might need altering schools table in the database and thus be related / can be done at the same time as https://github.com/Lokiedu/libertysoil-site/issues/728

This is a top priority item for us at this point. We need to showcase data available and capture imagination of people doing mashups and quick simple products, apps.

**Comment by indeyets** [comment]
Data is published to https://github.com/Lokiedu/opendata

It is not deployed to the production server yet. We'll do it once migration to the new VM is finished
